tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectWithInterfaceTest.php

```diff
@@ -27,11 +28,14 @@ public function setUp(): void
 </database>
 EOF;
             QuickBuilder::buildSchema($schema);
-            }
-      }
+        }
+    }
 
-      public function testClassHasInterface()
-      {
-            $this->assertInstanceOf('Foo\MyInterface', new \Foo\MyClassWithInterface());
-      }
+    /**
+     * @return void
+     */
+    public function testClassHasInterface()
+    {
+          $this->assertInstanceOf('Foo\MyInterface', new \Foo\MyClassWithInterface());
```

Comment:
Can we put this to the use section?

Reply:
Next sniffer run, there are more to come.